src/otx/cli/tools/train.py

```diff
@@ -199,6 +200,7 @@ def train(exit_stack: Optional[ExitStack] = None):  # pylint: disable=too-many-b
     args, override_param = get_args()
 
     config_manager = ConfigManager(args, workspace_root=args.workspace, mode=mode)
+    config_logger(config_manager.output_path / "otx.log", "INFO")
```

Comment:
How about adding the detailed path? If user conduct `train->eval->export->eval`, then it seems that user can't check the train log`. Also same opinion for the eval, explain, ...
```suggestion
    config_logger(config_manager.output_path / "otx_train.log", "INFO")
```

Reply:
log file is made in each output directory as below. I think it's unnecessary to have different name. How do you think about?
![image](https://github.com/openvinotoolkit/training_extensions/assets/92974184/7bb52709-9fe5-4acf-8956-0f83e5a535a3)

Reply:
Ah, if so I don't have any concerns :)